nixos/modules/services/web-apps/filebrowser.nix

```
@@ -0,0 +1,181 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) generators types hasPrefix literalExpression mkEnableOption mkIf
+                mkOption mkPackageOption optional optionalAttrs;
+
+  cfg = config.services.filebrowser;
+  enableDynamicUser =
+    assert cfg.user != "filebrowser" -> cfg.group != "filebrowser";
+    cfg.user == "filebrowser";
+  enableTls =
+    assert cfg.tlsCertificate != null -> cfg.tlsCertificateKey != null;
+    cfg.tlsCertificate != null;
+
+  configFile = pkgs.writeText "filebrowser-config.json" (generators.toJSON {} ({
+      database = "/var/lib/filebrowser/filebrowser.db";
```

We use 2 spaces to intend nix code

```suggestion
    database = "/var/lib/filebrowser/filebrowser.db";
```